<ui version="4.0" >
 <class>udg::QueryScreenBase</class>
 <widget class="QDialog" name="udg::QueryScreenBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Query Study</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../main/main.qrc" >:/images/pacsQuery.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="udg::QBasicSearchWidget" native="1" name="m_qbasicSearchWidget" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="udg::QPacsList" native="1" name="m_PACSNodes" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>150</width>
         <height>243</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="udg::QAdvancedSearchWidget" native="1" name="m_qadvancedSearchWidget" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_advancedSearchToolButton" >
       <property name="text" >
        <string>Advanced Search >></string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/search.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>211</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_clearToolButton" >
       <property name="text" >
        <string>Clear Search Fields</string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/clear.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_searchButton" >
       <property name="text" >
        <string>Search</string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/advancedSearch.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="m_tab" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="tabPosition" >
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <widget class="QWidget" name="m_tabCache" >
      <attribute name="title" >
       <string>Local Database</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../main/main.qrc" >:/images/database.png</iconset>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QSplitter" name="m_StudyTreeSeriesListQSplitter" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="childrenCollapsible" >
          <bool>false</bool>
         </property>
         <widget class="udg::QStudyTreeWidget" native="1" name="m_studyTreeWidgetCache" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>7</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="udg::QSeriesListWidget" native="1" name="m_seriesListWidgetCache" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>7</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_viewButtonLocal" >
           <property name="text" >
            <string>View</string>
           </property>
           <property name="icon" >
            <iconset resource="../main/main.qrc" >:/images/view.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabPacs" >
      <attribute name="title" >
       <string>PACS Server</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../main/main.qrc" >:/images/pacsServer.png</iconset>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="udg::QStudyTreeWidget" native="1" name="m_studyTreeWidgetPacs" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_viewButtonPACS" >
           <property name="text" >
            <string>View</string>
           </property>
           <property name="icon" >
            <iconset resource="../main/main.qrc" >:/images/view.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_retrieveButtonPACS" >
           <property name="text" >
            <string>Retrieve</string>
           </property>
           <property name="icon" >
            <iconset resource="../main/main.qrc" >:/images/retrieve.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabDicomdir" >
      <attribute name="title" >
       <string>DICOMDIR</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../main/main.qrc" >:/images/createDICOMDIR.png</iconset>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="udg::QStudyTreeWidget" native="1" name="m_studyTreeWidgetDicomdir" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_viewButtonDICOMDIR" >
           <property name="text" >
            <string>View</string>
           </property>
           <property name="icon" >
            <iconset resource="../main/main.qrc" >:/images/view.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_retrieveButtonDICOMDIR" >
           <property name="text" >
            <string>Retrieve</string>
           </property>
           <property name="icon" >
            <iconset resource="../main/main.qrc" >:/images/retrieve.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_operationListToolButton" >
       <property name="text" >
        <string>Operation List</string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/operationList.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="m_labelOperation" >
         <property name="text" >
          <string>Operation in progress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_operationAnimation" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>27</width>
         <height>42</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_showPACSNodesToolButton" >
       <property name="text" >
        <string>PACS Nodes</string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/pacsNodes.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_createDICOMDIRToolButton" >
       <property name="text" >
        <string>Create DICOMDIR</string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/createDICOMDIR.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>udg::QPacsList</class>
   <extends>QWidget</extends>
   <header>qpacslist.h</header>
  </customwidget>
  <customwidget>
   <class>udg::QSeriesListWidget</class>
   <extends>QWidget</extends>
   <header>qserieslistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>udg::QStudyTreeWidget</class>
   <extends>QWidget</extends>
   <header>qstudytreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_patientNameText</tabstop>
  <tabstop>m_patientIDText</tabstop>
  <tabstop>m_anyDateRadioButton</tabstop>
  <tabstop>m_todayRadioButton</tabstop>
  <tabstop>m_yesterdayRadioButton</tabstop>
  <tabstop>m_lastWeekRadioButton</tabstop>
  <tabstop>m_customDateRadioButton</tabstop>
  <tabstop>m_fromDateCheck</tabstop>
  <tabstop>m_fromStudyDate</tabstop>
  <tabstop>m_toDateCheck</tabstop>
  <tabstop>m_toStudyDate</tabstop>
  <tabstop>m_checkAll</tabstop>
  <tabstop>m_checkCR</tabstop>
  <tabstop>m_checkES</tabstop>
  <tabstop>m_checkNM</tabstop>
  <tabstop>m_checkRF</tabstop>
  <tabstop>m_checkXA</tabstop>
  <tabstop>m_checkCT</tabstop>
  <tabstop>m_checkMG</tabstop>
  <tabstop>m_checkDT</tabstop>
  <tabstop>m_checkSC</tabstop>
  <tabstop>m_checkOtherModality</tabstop>
  <tabstop>m_textOtherModality</tabstop>
  <tabstop>m_checkDX</tabstop>
  <tabstop>m_checkMR</tabstop>
  <tabstop>m_checkPT</tabstop>
  <tabstop>m_checkUS</tabstop>
  <tabstop>m_qwidgetAdvancedSearch</tabstop>
  <tabstop>m_studyTimeText</tabstop>
  <tabstop>m_studyUIDText</tabstop>
  <tabstop>m_studyIDText</tabstop>
  <tabstop>m_accessionNumberText</tabstop>
  <tabstop>m_studyModalityText</tabstop>
  <tabstop>m_referringPhysiciansNameText</tabstop>
  <tabstop>m_seriesUIDText</tabstop>
  <tabstop>m_requestedProcedureIDText</tabstop>
  <tabstop>m_seriesNumberText</tabstop>
  <tabstop>m_scheduledProcedureStepIDText</tabstop>
  <tabstop>m_PPStartTimeText</tabstop>
  <tabstop>m_PPStartDateText</tabstop>
  <tabstop>m_SOPInstanceUIDText</tabstop>
  <tabstop>m_instanceNumberText</tabstop>
  <tabstop>m_advancedSearchToolButton</tabstop>
  <tabstop>m_clearToolButton</tabstop>
  <tabstop>m_searchButton</tabstop>
  <tabstop>m_tab</tabstop>
  <tabstop>m_viewButtonLocal</tabstop>
  <tabstop>m_viewButtonPACS</tabstop>
  <tabstop>m_retrieveButtonPACS</tabstop>
  <tabstop>m_viewButtonDICOMDIR</tabstop>
  <tabstop>m_retrieveButtonDICOMDIR</tabstop>
  <tabstop>m_operationListToolButton</tabstop>
  <tabstop>m_showPACSNodesToolButton</tabstop>
  <tabstop>m_createDICOMDIRToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="../main/main.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_fromDateCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_fromStudyDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>211</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel" >
     <x>319</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_toDateCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_toStudyDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>195</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel" >
     <x>260</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customDateRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_fromDateCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>320</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel" >
     <x>211</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customDateRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_toDateCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel" >
     <x>211</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customDateRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_fromDateCheck</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>320</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel" >
     <x>211</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_customDateRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_toDateCheck</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>320</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel" >
     <x>195</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkOtherModality</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_textOtherModality</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>548</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel" >
     <x>579</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
